docs/licenses.md

```diff
@@ -0,0 +1,80 @@
+code regarding distributed binaries, and the distributed binaries does not contain LGPL3 code, therefore the code base does
+not need to be licensed under LGPL3.  
+In addition, precaution measures where taken to notify the user about the LGPL3 license, and the consequences of building this 
+code base with LGPL3 components. Therefore, it's the end user's responsibility to distribute this code base and the binaries 
+that he compiled under the LGPL3 license, we do not take any responsibility, nor accountability on the end user's choices and actions, 
+we just give him the freedom to choose if he is going to use LGPL3 components.
+
+Thirdly, the original libraries code is left unmodified, therefore the code modification related sections are irrelevant.
+
+In conclusion, all the copyright notices and LGPL3 notices were provided to the users, and the OPT-IN mechanism makes sure
+that it's the end users choice to use LGPL3 components and not the maintainers choice. Therefore, this code base is distributed
+under MIT license while being compatible with the restrictions of the LGPL3 license.
+
+### Personal Notes on the LGPL3 and GPL3 licenses
+There is some hypocrisy among the GNU community, while they fight for user's "freedom" and "protect" it, the end result
+is the complete opposite freedom is taken both from the end users and from the opensource code developers.  
+
+For the end user the freedom is taken by the form of less features available on the distriputed software, because there are 
+certain features that come from libraries/source code that are incompatible with GNU license so when recieving a GNU protected
+binary (and vice versa), you have less features, and if you want the lost features back, you need to compile the software from 
+source, without the GNU protected parts which can sometimes be very tidious and praticly imposiable for novice users.  
+In example, the git openssl backend is not available when git is linked with GNU, because openSSL is incompatible with GNU.
+Forcing the end user to compile git himself (not an easy task) and this will occur in any GNU based linux distribution which is 
+insane. In contrast using Apple's propriety software gives the user better experience because there are no GNU shenanigans.  
+
+For the developer the rapist conditions that the GNU license imposes leaves the developer with 3 unattractive options:
+The first option is to ignore any GPL protected code, thus limiting his available options to incorporate from the open source 
+community. So his freedom is taken by limiting the available options which are in the firstplace shared so the community/humanity
+can enjoy the code/solution that have already been written.  
+
+The second option that the devloper has is to surrender to GPL's license thus making his entire code protected by GPL's license
+and now he is raping his users to use GPL's license.  
```

Comment:
If you’re developing free software why is that even a problem?

If you’re not, than I don’t give a fuck about your right to develop closed source code.